scripts/deploy.sh

```diff
@@ -12,28 +12,50 @@ source scripts/cleanup.sh
 
 # ---- On Deployment -----
 #
-# This is run on pushes to master, or release/**
+# This finds matching entries in packages/hoprd/releases.json and deploys accordingly
 #
 # ENV Variables:
 # - GITHUB_REF: ie. `/refs/heads/mybranch`
 # - FUNDING_PRIV_KEY: funding private key, raw
 # - BS_PASSWORD: database password
 
-# Get version from package.json if not already set
-if [ -z "${RELEASE:-}" ]; then
-  RELEASE=$(node -p -e "require('./packages/hoprd/package.json').version")
-fi
-
-# Get RELEASE_NAME, from environment
-get_environment
-
-TESTNET_NAME="$RELEASE_NAME-$(echo "$VERSION_MAJ_MIN" | sed 's/\./-/g')"
-TESTNET_SIZE=3
-
-echo "Cleaning up before deploy"
-#cleanup
-
-cleanup_instance "${TESTNET_NAME}"
-
-echo "Starting testnet '$TESTNET_NAME' with $TESTNET_SIZE nodes and image hoprd:$RELEASE, environment id: $ENVIRONMENT_ID"
-start_testnet $TESTNET_NAME $TESTNET_SIZE "gcr.io/hoprassociation/hoprd:$RELEASE" $ENVIRONMENT_ID
+_jq() {
+  echo "$1" | base64 --decode | jq -r "$2"
+}
+
+echo "Looking for releases to deploy (GITHUB_REF == ${GITHUB_REF})"
+
+# iterate through releases with git_ref == $GITHUB_REF
+for row in $(cat packages/hoprd/releases.json | ./node_modules/.bin/strip-json-comments | jq -r ".[] | select(.git_ref==\"${GITHUB_REF}\") | @base64"); do
+  declare RELEASE_ID=$(_jq "${row}" ".id")
```

Comment:
We need to decide whether we want to use lower or upper case variable names. The scripts I've touched all use lower-case to distinguish from env vars.